app/modules/ads/ml/cheshire-cat.js

```diff
@@ -85,23 +84,20 @@ function getBtlSlotStatus(btlStatus, callId, fallbackStatus) {
   return slotStatus;
 }
 
-export const billTheLizardWrapper = {
-  configureBillTheLizard(billTheLizardConfig) {
+export const cheshireCat = {
+  configureCheshireCat(billTheLizardConfig) {
```

Comment:
Also, since the object is called `cheshireCat`, then there is no need to repeat the name in method name. `configure` would be just enough.

Reply:
👍